Userland/Libraries/LibWeb/Streams/AbstractOperations.cpp

```diff
@@ -7,16 +7,18 @@
 
 #include <LibWeb/Streams/AbstractOperations.h>
 #include <LibWeb/Streams/ReadableStream.h>
+#include <LibWeb/Streams/ReadableStreamDefaultReader.h>
 #include <LibWeb/Streams/ReadableStreamGenericReader.h>
 #include <LibWeb/WebIDL/ExceptionOr.h>
+#include <LibWeb/WebIDL/Promise.h>
 
 namespace Web::Streams {
 
 // https://streams.spec.whatwg.org/#is-readable-stream-locked
 bool is_readable_stream_locked(ReadableStream const& stream)
 {
     // 1. If stream.[[reader]] is undefined, return false.
-    if (stream.reader() == nullptr)
+    if (!stream.reader())
```

Comment:
No fixups across commits

Reply:
Sorry, don't quite understand this one, this is the first change I made to this function in this PR. It's not really required but this is how I do all of the other similar checks in the PR, so I thought it made sense to change. Is it worth it's own commit?

Reply:
oh, I was 99% sure this was new code added in one of the previous commits, maybe I confused it with something else. Never mind then :)